ckan/config/environment.py

```diff
@@ -224,6 +225,16 @@ def update_config():
             'ckan.site_url should be a full URL, including the schema '
             '(http or https)')
 
+    display_timezone = config.get('ckan.display_timezone', '')
+    if display_timezone:
+        if (
+            display_timezone != 'server' and
+            display_timezone not in pytz.all_timezones
+        ):
+            raise RuntimeError(
+                'ckan.display_timezone is empty or not a valid timezone'
```

Comment:
Just trying to follow what other similar errors use buit I don't mind adding a new exception